ports/servoshell/lib.rs

```diff
@@ -61,11 +61,6 @@ pub fn main() {
     // Parse the command line options and store them globally
     let args: Vec<String> = env::args().collect();
     let mut opts = Options::new();
-    opts.optflag(
```

Comment:
Is this change supposed to be part of this PR?

Reply:
No, this is from another change. I've gone ahead and removed it from this one.